docs/observability-logging-tracking.md

```
@@ -0,0 +1,109 @@
+| [#122](https://github.com/SaplingLearn/Sapling/issues/122) | Admin analytics dashboard UI (charts/visualizations) | Jose | Frontend | #121 |
+
+### Load (even split — 2 issues each)
+
+- **Luke** (Backend): #115, #116
+- **Andres** (Fullstack): #117, #120
+- **Jack** (AI Engineer): #118, #119
+- **Jose** (Frontend): #121, #122
+
+## Dependency graph
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)